src/components/Calculator.js

```diff
@@ -1,53 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import calculate from '../logic/calculate';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.calculating = this.calculating.bind(this);
-    this.state = {
-      total: props.start,
-    };
-  }
+const Calculator = ({ start }) => {
+  const [display, setDisplay] = useState({ next: start });
+  const onClick = (e) => {
+    setDisplay((display, calculate(display, e.target.textContent)));
+  };
```

Comment:
:heavy_check_mark: Great job in refactoring the Calculator component. :+1: After the refactoring, your App still works as expected. :clap: 

[OPTIONAL]

- [ ] Please, consider handling the browser error that's generated when a user double clicks an operator. I strongly recommend that you do this.